src/shared/GmdVizPanel/components/OpenAssistant.tsx

```diff
@@ -0,0 +1,89 @@
+import { css, cx } from '@emotion/css';
+import { createAssistantContextItem, isAssistantAvailable, openAssistant } from '@grafana/assistant';
+import { type GrafanaTheme2 } from '@grafana/data';
+import { sceneGraph, SceneObjectBase, VizPanel, type SceneComponentProps, type SceneObjectState } from '@grafana/scenes';
+import { Button, useStyles2 } from '@grafana/ui';
+import React, { useEffect, useState } from 'react';
+
+import { getPanelData } from './addToDashboard/addToDashboard';
+
+const OPEN_ASSISTANT_LABEL = 'Open in Assistant';
+
+interface OpenAssistantState extends SceneObjectState {}
+
+export class OpenAssistant extends SceneObjectBase<OpenAssistantState> {
+  constructor() {
+    super({});
+  }
+
+  public static readonly Component = ({ model }: SceneComponentProps<OpenAssistant>) => {
+    const styles = useStyles2(getStyles);
+    const [isAvailable, setIsAvailable] = useState(false);
+
+    const vizPanel = sceneGraph.findObject(model, (o) => o instanceof VizPanel) as VizPanel | undefined;
+
+    useEffect(() => {
+      const subscription = isAssistantAvailable().subscribe((available) => {
+        setIsAvailable(available);
+      });
+      return () => subscription.unsubscribe();
+    }, []);
+
+    if (!isAvailable || !vizPanel) {
+      return null;
+    }
+
+    const handleClick = () => {
+      const { panel } = getPanelData(vizPanel);
+
+      // Get metric name from panel title
+      const metricName = panel.title || 'unknown';
+
+      // Extract the query expression and remove __ignore_usage__ label (same as ExploreAction)
+      const rawExpr = panel.targets?.[0]?.expr;
+      let query = typeof rawExpr === 'string' ? rawExpr : '';
+      if (query.includes('__ignore_usage__')) {
+        query = query.replace(/,?__ignore_usage__="",?/, '');
+      }
+
+      // Build prompt with or without query
+      const queryPart = query ? ` The current metrics drilldown query is: ${query}.` : '';
+
+      // Only include datasource context when we have a valid UID
+      const datasourceUid = panel.datasource?.uid;
+      const context = datasourceUid
+        ? [createAssistantContextItem('datasource', { datasourceUid })]
+        : [];
+
+      openAssistant({
+        origin: 'grafana-metricsdrilldown-app/metric-panel',
+        prompt: `Help me understand the metric "${metricName}" and explain what it measures.${queryPart} Be concise and to the point.`,
```

Comment:
Could you remove `Be concise and to the point.`. This does not need to be added as we are already including it in Assistant prompt.